content/backmatter.tex

```diff
@@ -972,52 +1000,52 @@ \section{Version 1.2}
       avoid confusion with \Fortran's \KEYWORD{target} keyword.
 %
 \item New Execution Model for exiting/finishing \openshmem programs.
-\\See Section  \ref{subsec:execution_model}.
+\ChangelogRef{subsec:execution_model}%
 %
 \item New library constants to support API that query version and name information.
-\\See Section \ref{subsec:library_constants}.
+\ChangelogRef{subsec:library_constants}%
 %
 \item New API \FUNC{shmem\_init} to provide mechanism to start an \openshmem
       program and replace deprecated \FUNC{start\_pes}.
-\\See Section \ref{subsec:shmem_init}.
+\ChangelogRef{subsec:shmem_init, dep:start_pes}%
 %
 \item Deprecation of \FUNC{\_my\_pe} and \FUNC{\_num\_pes} routines.
-\\See Sections \ref{subsec:shmem_my_pe} and \ref{subsec:shmem_n_pes}.
+\ChangelogRef{dep:func_not_shmemunder}%
 %
 \item New API \FUNC{shmem\_finalize} to provide collective mechanism to cleanly
       exit an \openshmem program and release resources.
-\\See Section \ref{subsec:shmem_finalize}.
+\ChangelogRef{subsec:shmem_finalize}%
 %
 \item New API \FUNC{shmem\_global\_exit} to provide mechanism to exit an
     \openshmem program.
-\\See Section \ref{subsec:shmem_global_exit}.
+\ChangelogRef{subsec:shmem_global_exit}%
 %
 \item Clarification related to the address of the referenced object in
     \FUNC{shmem\_ptr}.
-\\See Section \ref{subsec:shmem_ptr}.
+\ChangelogRef{subsec:shmem_ptr}%
 %
 \item New API to query the version and name information.
-\\See Section \ref{subsec:shmem_info_get_version} and \ref{subsec:shmem_info_get_name}.
+\ChangelogRef{subsec:shmem_info_get_version, subsec:shmem_info_get_name}%
 %
 \item \openshmem library API normalization. All \Cstd symmetric memory management
       API begins with  \FUNC{shmem\_}.
-\\See Section \ref{subsec:shfree}.
+\ChangelogRef{subsec:shfree, dep:func_not_shmemunder}%
 %
 \item Notes and clarifications added to \FUNC{shmem\_malloc}.
-\\See Section \ref{subsec:shfree}.
+\ChangelogRef{subsec:shfree}%
 %
 \item Deprecation of \Fortran API routine \FUNC{SHMEM\_PUT}.
-\\See Section \ref{subsec:shmem_put}.
+\ChangelogRef{dep:fortran_shmem_put}%
```

Comment:
Lost reference.

Reply:
This one feels dubious now that Fortran is removed. Should it still point to `subsec:shmem_put`?

Reply:
I left it in. It's at least historically accurate.

Reply:
That's a fair point. Once the interfaces are removed, the only meaningful thing to reference in the current document is the deprecation annex. What about adding a new macro for referencing across versions of the document, something like `\crossref{1.4}{9.5.1}` -> `OpenSHMEM 1.4, Section 9.5.1`?

Reply:
Resolved. I wrote it in directly instead of adding a new command because:
- I don't expect further removals now that Fortran and SHMEM_CACHE are removed.
- There's potentially a lot of variations that would be non-trivial to code for. E.g., a Sections range, the Annexes, or Tables, etc. Alas, `\cref` would be ideal if the old-doc references could be hardcoded...